includes/blocks/subtitle-block/block.json

```
@@ -0,0 +1,25 @@
+{
+	"name": "newspack-block-theme/subtitle",
```

```suggestion
	"name": "newspack-block-theme/article-subtitle",
```

NIT suggestion just to be extra safe since changing a block name, if we ever need to, can be cumbersome.

---

Done in 2ace418e951711c036f58a97c0847d44ffd6d00b